docs/toc/processor.rst

```diff
@@ -0,0 +1,95 @@
+Processor
+==========
+
+A pipeline component that wraps inference model and set up inference related work.
+
+
+
+Usage
+------
+
+* wrap model by setting `self.model`
+* set up inference related behaviors
+
+
+Functions
+------------------
+
+* `initialize()`: initilize model and inference related attributes.
+* `process()`: using loaded model to make predictions and write the prediction results out
```

Comment:
Again, `model`, `prediction`, `write prediction` are all not essential parts of `process`. The `process` function has access to the data pack, and the `process` function is called at certain stages during the pipeline. Those are the important information.

Reply:
Please try to understand the lifecycle of a pipeline, when are stuff called, and then come back to explain them.